app/src/main/java/com/example/evently/ui/organizer/OrganizerActivity.java

```diff
@@ -0,0 +1,12 @@
+package com.example.evently.ui.organizer;
+
+import com.example.evently.R;
+import com.example.evently.ui.common.ArchitectureActivity;
+
+public class OrganizerActivity extends ArchitectureActivity {
```

Comment:
You can make an `EntrantActivity` just like this but make sure to make your own `entrant_graph`